src/super_gradients/training/utils/callbacks.py

```diff
@@ -73,6 +74,11 @@ def __init__(self, epoch=None, batch_idx=None, optimizer=None, metrics_dict=None
         self.sg_logger = sg_logger
         self.train_loader = train_loader
         self.valid_loader = valid_loader
+        self.quant_modules_calib_method = quant_modules_calib_method
+        self.checkpoints_dir_path = checkpoints_dir_path
```

Comment:
here too

Reply:
fixed (also redundant quant fields that are passed to init of qat cllabcak) https://github.com/Deci-AI/super-gradients/pull/232/commits/f1391fbeee6114e182d83dc39676bf999a4d69d4